.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: generic
+
+env:
+    global:
+        - PYTHONIOENCODING=UTF8
+        - MPLBACKEND=Agg
+
+matrix:
+  include:
+    - os: linux
+      language: generic
+      env:
+        - PYTHON_VERSION=3.6
+        - CONDA=true
+
+    - os: linux
+      language: generic
+      env:
+        - PYTHON_VERSION=3.7
+        - CONDA=true
+
+before_install:
+
+    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    - bash miniconda.sh -b -p $HOME/miniconda
+    - . $HOME/miniconda/etc/profile.d/conda.sh
+    - hash -r
+    - conda config --set always_yes yes --set changeps1 no
+    - conda update -q conda  # get latest conda version
+    - conda info -a # Useful for debugging any issues with conda
+
+install:
+    - conda env create -f environment.yml
```

Comment:
This suffers from the same problem as lstchain and ctaplots: it does not use the python version you give in the `PYTHON_VERSION` environment variable.

I would suggest to just not use conda (testing with a pure pip install is fine and actually more general) or use the solution employed by ctapipe, lstchain and others, namely to replace the python version in environment.yml with sed

Reply:
I never actually realized this.
if it's not super urgent (namely using 3.7 or 3.8 - I guess it will use always the latest stable release that comes with conda base?) maybe we can fix this in another PR?
I would suggest to fix things generally for the moment, so we know that there is a base backup always working